<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>428</width>
    <height>296</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>150</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextUnderIcon</enum>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="mainWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::NoToolBarArea</set>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSettings"/>
   <addaction name="actionAccount"/>
   <addaction name="actionLog"/>
   <addaction name="actionAbout"/>
  </widget>
  <action name="actionSettings">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../resources.qrc">
     <normaloff>:/icons/switch.png</normaloff>:/icons/switch.png</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="toolTip">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionLog">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../resources.qrc">
     <normaloff>:/icons/log@2x.png</normaloff>:/icons/log@2x.png</iconset>
   </property>
   <property name="text">
    <string>Log</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../resources.qrc">
     <normaloff>:/icons/info@2x.png</normaloff>:/icons/info@2x.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="toolTip">
    <string>About</string>
   </property>
  </action>
  <action name="actionAccount">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../resources.qrc">
     <normaloff>:/icons/person@2x.png</normaloff>:/icons/person@2x.png</iconset>
   </property>
   <property name="text">
    <string>Account</string>
   </property>
   <property name="toolTip">
    <string>Account</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../resources.qrc"/>
 </resources>
 <connections/>
</ui>
